portfolio/src/main/webapp/index.html

```diff
@@ -14,14 +14,52 @@ <h1>Mauro Guerrero</h1>
       <a href="https://www.linkedin.com/in/mauro-guerrero-286b69172"><i class="fa fa-linkedin-square"></i></a>
       <a href="https://github.com/MauroGuerrero20"><i class="fa fa-github"></i></a>
     </div>
-    <div id="content">
-      <!--About me-->
+    <!--About me-->
+    <div id="about_me">
       <h2>About me:<img class="profile_image" src="images/profile_pic.jpg" alt="Avatar"/></h2>
-      <p>Hi, I am a Texas A&ampM Engineering Student that is currently pursuing a B.A. in Computer Engineering.
+      <p>Hi, I am a Texas A&ampM Engineering Student that is currently pursuing a B.A. in Computer
+        Engineering.
       </p>
       <p>Click here to get a random message:</p>
       <button id="randBtn">Click me</button>
       <div id="quote-container"></div>
     </div>
+    <!--Work Exp-->
+    <div class="work_background">
+      <div id="work_exp">
+        <h2>Work Experience</h2>
+        <div class="logo">
+          <img class="google_logo" src="images/google_logo.jpg">
+        </div>
+        <div>
+          <p>
+            <br>
+          <h3>Engineering Practicum Intern, Sunnyvale, CA<br>Summer 2019</h3>
+          <li>Security based internship, worked on developing the Test Identity Provider's (Idp)
+            metadata
```

Comment:
nit: can you keep "metadata" + "using" + "the OpenSAML..." lines together on a single line? Same for the text below. As per Google's official HTML style guide, there's no hard character limit for lines in HTML.